front/temporal/tracker/activities.ts

```diff
@@ -52,6 +64,11 @@ export async function trackersGenerationActivity(
   documentHash: string,
   dataSourceConnectorProvider: ConnectorProvider | null
 ) {
+  if (Context.current().info.attempt > 1) {
+    // TODO(DOC_TRACKER): mechanism to retry "manually"
+    throw new Error("Too many attempts");
```

Comment:
little safety, I don't want to retry these indefinitely because of the cost associated, and I also don't want it to be lost.